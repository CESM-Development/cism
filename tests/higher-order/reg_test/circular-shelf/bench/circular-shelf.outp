 (t_initf) Read in prof_inparm namelist from: perf_in
 (t_initf) Using profile_disable= F  profile_timer=           1
 (t_initf)  profile_depth_limit=       99999  profile_detail_limit=           1
 (t_initf)  profile_barrier= F  profile_outpe_num=           0
 (t_initf)  profile_outpe_stride=           1  profile_single_file= F
 (t_initf)  profile_global_stats= T  profile_papi_enable= F
 Layout(EW,NS) =           41          21  total procs =            1
 Layout(EW,NS) =           41          20  total procs =            1
  
 Compute higher-order ice velocities, time =   0.0000000000000000     
 Using GlobalIDs...

Partition Info in init_trilinos: Total nodes = 7308  Max = 3906  Min = 3402  Ave = 3654
  
 Running Payne/Price higher-order dynamics solver
  
 iter #     resid (L2 norm)       target resid
  
Precon Linear Solve Succeeded: 6.19e-09 drop in 92 its (avg: 92 its/slv, 100% success)
Precon Linear Solve Succeeded: 9.82e-09 drop in 90 its (avg: 91 its/slv, 100% success)
   1         37.3466            0.100000E-03
Precon Linear Solve Succeeded: 6.26e-09 drop in 88 its (avg: 90 its/slv, 100% success)
Precon Linear Solve Succeeded: 6.79e-09 drop in 86 its (avg: 89 its/slv, 100% success)
   2         37.3078            0.100000E-03
Precon Linear Solve Succeeded: 6.65e-09 drop in 88 its (avg: 88.8 its/slv, 100% success)
Precon Linear Solve Succeeded: 6e-09 drop in 86 its (avg: 88.3 its/slv, 100% success)
   3         37.2163            0.100000E-03
Precon Linear Solve Succeeded: 9.2e-09 drop in 87 its (avg: 88.1 its/slv, 100% success)
Precon Linear Solve Succeeded: 9.27e-09 drop in 86 its (avg: 87.9 its/slv, 100% success)
   4         35.5186            0.100000E-03
Precon Linear Solve Succeeded: 8.45e-09 drop in 88 its (avg: 87.9 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.26e-09 drop in 87 its (avg: 87.8 its/slv, 100% success)
   5         32.0032            0.100000E-03
Precon Linear Solve Succeeded: 8.31e-09 drop in 86 its (avg: 87.6 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.82e-09 drop in 87 its (avg: 87.6 its/slv, 100% success)
   6         26.9310            0.100000E-03
Precon Linear Solve Succeeded: 5.52e-09 drop in 89 its (avg: 87.7 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.66e-09 drop in 87 its (avg: 87.6 its/slv, 100% success)
   7         21.1717            0.100000E-03
Precon Linear Solve Succeeded: 6.63e-09 drop in 89 its (avg: 87.7 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.17e-09 drop in 87 its (avg: 87.7 its/slv, 100% success)
   8         15.6199            0.100000E-03
Precon Linear Solve Succeeded: 7.07e-09 drop in 89 its (avg: 87.8 its/slv, 100% success)
Precon Linear Solve Succeeded: 9.58e-09 drop in 86 its (avg: 87.7 its/slv, 100% success)
   9         10.8986            0.100000E-03
Precon Linear Solve Succeeded: 7.03e-09 drop in 89 its (avg: 87.7 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.99e-09 drop in 86 its (avg: 87.7 its/slv, 100% success)
  10         7.27955            0.100000E-03
Precon Linear Solve Succeeded: 6.83e-09 drop in 89 its (avg: 87.7 its/slv, 100% success)
Precon Linear Solve Succeeded: 9.91e-09 drop in 85 its (avg: 87.6 its/slv, 100% success)
  11         4.72141            0.100000E-03
Precon Linear Solve Succeeded: 6.64e-09 drop in 89 its (avg: 87.7 its/slv, 100% success)
Precon Linear Solve Succeeded: 8.1e-09 drop in 85 its (avg: 87.5 its/slv, 100% success)
  12         3.00996            0.100000E-03
Precon Linear Solve Succeeded: 6.51e-09 drop in 89 its (avg: 87.6 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.73e-09 drop in 85 its (avg: 87.5 its/slv, 100% success)
  13         1.90101            0.100000E-03
Precon Linear Solve Succeeded: 6.43e-09 drop in 89 its (avg: 87.6 its/slv, 100% success)
Precon Linear Solve Succeeded: 8.23e-09 drop in 85 its (avg: 87.5 its/slv, 100% success)
  14         1.19418            0.100000E-03
Precon Linear Solve Succeeded: 6.38e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 8.91e-09 drop in 85 its (avg: 87.4 its/slv, 100% success)
  15        0.730101            0.100000E-03
Precon Linear Solve Succeeded: 6.34e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 9.5e-09 drop in 85 its (avg: 87.4 its/slv, 100% success)
  16        0.476198            0.100000E-03
Precon Linear Solve Succeeded: 6.32e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 6.89e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  17        0.292959            0.100000E-03
Precon Linear Solve Succeeded: 6.31e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.13e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  18        0.182557            0.100000E-03
Precon Linear Solve Succeeded: 6.3e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.29e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  19        0.112932            0.100000E-03
Precon Linear Solve Succeeded: 6.3e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.39e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  20        0.698657E-01        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.47e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  21        0.433318E-01        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.53e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  22        0.267015E-01        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.57e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  23        0.167561E-01        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.6e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  24        0.103606E-01        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.62e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  25        0.648913E-02        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.64e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  26        0.413556E-02        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.65e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  27        0.265044E-02        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.66e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  28        0.174715E-02        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.66e-09 drop in 86 its (avg: 87.4 its/slv, 100% success)
  29        0.116584E-02        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.67e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  30        0.806092E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.67e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  31        0.556200E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.67e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  32        0.400877E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.67e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  33        0.287058E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.67e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  34        0.211027E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.67e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  35        0.157997E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.68e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  36        0.118403E-03        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.68e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  37        0.890373E-04        0.100000E-03
Precon Linear Solve Succeeded: 6.29e-09 drop in 89 its (avg: 87.5 its/slv, 100% success)
Precon Linear Solve Succeeded: 7.68e-09 drop in 86 its (avg: 87.5 its/slv, 100% success)
  38        0.700377E-04        0.100000E-03
  
 Writing diagnostics to log file, time(yr) =   0.0000000000000000     
 idiag_global, jdiag_global:           1           1
 local rank, i, j:           0           3           3
Application 14537984 resources: utime ~34s, stime ~0s Rss ~32764 inblocks ~33118 outblocks ~89475
