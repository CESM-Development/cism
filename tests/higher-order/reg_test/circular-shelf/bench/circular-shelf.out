 (t_initf) Read in prof_inparm namelist from: perf_in
 (t_initf) Using profile_disable= F  profile_timer=           1
 (t_initf)  profile_depth_limit=       99999  profile_detail_limit=           1
 (t_initf)  profile_barrier= F  profile_outpe_num=           0
 (t_initf)  profile_outpe_stride=           1  profile_single_file= F
 (t_initf)  profile_global_stats= T  profile_papi_enable= F
 Layout(EW,NS) =           41          21  total procs =            1
 Layout(EW,NS) =           41          20  total procs =            1
  
 Compute higher-order ice velocities, time =   0.0000000000000000     
 Using GlobalIDs...

Partition Info in init_trilinos: Total nodes = 7308  Max = 3906  Min = 3402  Ave = 3654
  
 Running Payne/Price higher-order dynamics with JFNK solver
NOXINIT CALLED    for nelem=7812
NOXInit: param list is: (delete this debug line)
Jacobian Operator = Matrix-Free   [unused]
Matrix-Free Perturbation = 0.0001   [unused]
Lean Matrix Free = 1   [unused]
NOX -> 
 Nonlinear Solver = Line Search Based   [unused]
 Direction -> 
  Method = Newton   [unused]
  Newton -> 
   Forcing Term Method = Type 2   [unused]
   Rescue Bad Newton Solve = 1   [unused]
   Stratimikos Linear Solver -> 
    NOX Stratimikos Options -> 
     [empty list]
    Stratimikos -> 
     Linear Solver Type = Belos   [unused]
     Preconditioner Type = None   [unused]
     Linear Solver Types -> 
      Belos -> 
       Solver Type = Block GMRES   [unused]
       Solver Types -> 
        Block GMRES -> 
         Convergence Tolerance = 0.0001   [unused]
         Output Frequency = 1   [unused]
         Output Style = 1   [unused]
         Verbosity = 33   [unused]
         Maximum Iterations = 100   [unused]
         Block Size = 1   [unused]
         Num Blocks = 100   [unused]
         Flexible Gmres = 1   [unused]
 Line Search -> 
  [empty list]
 Printing -> 
  Output Information -> 
   Error = 1   [unused]
   Warning = 1   [unused]
   Outer Iteration = 1   [unused]
   Parameters = 0   [unused]
   Details = 0   [unused]
   Linear Solver Details = 0   [unused]
 Status Tests -> 
  Test Type = Combo   [unused]
  Combo Type = OR   [unused]
  Number of Tests = 2   [unused]
  Test 0 -> 
   Test Type = NormF   [unused]
   Tolerance = 0.0001   [unused]
  Test 1 -> 
   Test Type = MaxIters   [unused]
   Maximum Iterations = 30   [unused]

NOXSolve called

************************************************************************
-- Nonlinear Solver Step 0 -- 
||F|| = 3.735e+01  step = 0.000e+00  dx = 0.000e+00
************************************************************************

 Precon Linear Solve Succeeded: 6.15e-09 drop in 92 its (avg: 92 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.99e-09 drop in 90 its (avg: 91 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.7e-09 drop in 93 its (avg: 91.7 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.99e-09 drop in 90 its (avg: 91.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.84e-09 drop in 91 its (avg: 91.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.2e-09 drop in 89 its (avg: 90.8 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.13e-09 drop in 91 its (avg: 90.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.45e-09 drop in 90 its (avg: 90.8 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 4 iterations with total CPU time of 1.82394 sec

************************************************************************
-- Nonlinear Solver Step 1 -- 
||F|| = 3.730e+01  step = 1.000e+00  dx = 3.682e-06
************************************************************************

 Precon Linear Solve Succeeded: 6.51e-09 drop in 88 its (avg: 90.4 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.25e-09 drop in 86 its (avg: 90 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 1 iterations with total CPU time of 0.427086 sec

************************************************************************
-- Nonlinear Solver Step 2 -- 
||F|| = 3.682e+01  step = 1.000e+00  dx = 3.352e-03
************************************************************************

 Precon Linear Solve Succeeded: 6.75e-09 drop in 88 its (avg: 89.8 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.37e-09 drop in 86 its (avg: 89.5 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 1 iterations with total CPU time of 0.42418 sec

************************************************************************
-- Nonlinear Solver Step 3 -- 
||F|| = 3.434e+01  step = 1.000e+00  dx = 4.912e-01
************************************************************************

 Precon Linear Solve Succeeded: 6.53e-09 drop in 88 its (avg: 89.4 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.54e-09 drop in 86 its (avg: 89.1 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 1 iterations with total CPU time of 0.427063 sec

************************************************************************
-- Nonlinear Solver Step 4 -- 
||F|| = 2.783e+01  step = 1.000e+00  dx = 1.304e+01
************************************************************************

 Precon Linear Solve Succeeded: 5.59e-09 drop in 88 its (avg: 89.1 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.39e-09 drop in 86 its (avg: 88.9 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 1 iterations with total CPU time of 0.427647 sec

************************************************************************
-- Nonlinear Solver Step 5 -- 
||F|| = 1.765e+01  step = 1.000e+00  dx = 8.075e+01
************************************************************************

 Precon Linear Solve Succeeded: 8.78e-09 drop in 86 its (avg: 88.7 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.45e-09 drop in 84 its (avg: 88.4 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 1 iterations with total CPU time of 0.41063 sec

************************************************************************
-- Nonlinear Solver Step 6 -- 
||F|| = 7.067e+00  step = 1.000e+00  dx = 1.113e+02
************************************************************************

 Precon Linear Solve Succeeded: 6.31e-09 drop in 87 its (avg: 88.4 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.2e-09 drop in 84 its (avg: 88.2 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 1 iterations with total CPU time of 0.414429 sec

************************************************************************
-- Nonlinear Solver Step 7 -- 
||F|| = 1.539e+00  step = 1.000e+00  dx = 2.091e+01
************************************************************************

 Precon Linear Solve Succeeded: 9.91e-09 drop in 83 its (avg: 87.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 5.98e-09 drop in 84 its (avg: 87.7 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.42e-09 drop in 86 its (avg: 87.7 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8e-09 drop in 83 its (avg: 87.5 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 2 iterations with total CPU time of 0.808655 sec

************************************************************************
-- Nonlinear Solver Step 8 -- 
||F|| = 1.575e-01  step = 1.000e+00  dx = 6.062e-01
************************************************************************

 Precon Linear Solve Succeeded: 7.3e-09 drop in 88 its (avg: 87.5 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.33e-09 drop in 85 its (avg: 87.4 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.68e-09 drop in 86 its (avg: 87.3 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.36e-09 drop in 85 its (avg: 87.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.1e-09 drop in 88 its (avg: 87.3 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.27e-09 drop in 86 its (avg: 87.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.68e-09 drop in 87 its (avg: 87.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.85e-09 drop in 85 its (avg: 87.2 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 4 iterations with total CPU time of 1.68038 sec

************************************************************************
-- Nonlinear Solver Step 9 -- 
||F|| = 4.252e-03  step = 1.000e+00  dx = 2.214e-01
************************************************************************

 Precon Linear Solve Succeeded: 5.82e-09 drop in 88 its (avg: 87.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.77e-09 drop in 87 its (avg: 87.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.72e-09 drop in 87 its (avg: 87.2 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.08e-09 drop in 85 its (avg: 87.1 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.83e-09 drop in 87 its (avg: 87.1 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.6e-09 drop in 85 its (avg: 87.1 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.64e-09 drop in 87 its (avg: 87.1 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.36e-09 drop in 85 its (avg: 87 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.08e-09 drop in 86 its (avg: 87 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.99e-09 drop in 86 its (avg: 87 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.61e-09 drop in 87 its (avg: 87 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.45e-09 drop in 83 its (avg: 86.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.97e-09 drop in 88 its (avg: 86.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.5e-09 drop in 87 its (avg: 86.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 6.72e-09 drop in 88 its (avg: 86.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 7.22e-09 drop in 86 its (avg: 86.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 8.48e-09 drop in 88 its (avg: 86.9 its/slv, 100% success)
 Precon Linear Solve Succeeded: 9.52e-09 drop in 85 its (avg: 86.9 its/slv, 100% success)
 
 The Belos solver of type "Belos::BlockGmresSolMgr<...,double>{Variant='Flexible', Ortho Type='DGKS', Block Size=1, Num Blocks=100, Max Restarts=20}" returned a solve status of "SOLVE_STATUS_CONVERGED" in 9 iterations with total CPU time of 3.81179 sec

************************************************************************
-- Nonlinear Solver Step 10 -- 
||F|| = 1.058e-05  step = 1.000e+00  dx = 2.490e-02 (Converged!)
************************************************************************

************************************************************************
-- Final Status Test Results --
Converged....OR Combination -> 
  Converged....F-Norm = 1.058e-05 < 1.000e-04
               (Unscaled Two-Norm, Absolute Tolerance)
  ??...........Number of Iterations = -1 < 30
************************************************************************
Step Converged
Convergence Stats: for step  #1 : Newton, Krylov, Kr/Ne; LastKrylov, LastTol: 10  25  2.5  9 0.00198
NOXFinish called
  
 Writing diagnostics to log file, time(yr) =   0.0000000000000000     
 idiag_global, jdiag_global:           1           1
 local rank, i, j:           0           3           3
Application 14537980 resources: utime ~23s, stime ~0s Rss ~40504 inblocks ~33129 outblocks ~89481
